pepper-apis/dsm-core/src/main/java/org/broadinstitute/lddp/security/Auth0Util.java

```diff
@@ -60,19 +60,23 @@ public Auth0Util(@NonNull String account, @NonNull List<String> connections, boo
         this.audience = audience;
     }
 
-    public Auth0UserInfo getAuth0UserInfo(@NonNull String idToken) {
-        Map<String, Claim> auth0Claims = verifyAndParseAuth0TokenClaims(idToken);
-        boolean isEmailVerified = false;
-        Claim emailVerifiedClaim = auth0Claims.getOrDefault("email_verified", null);
-        if (emailVerifiedClaim != null && !emailVerifiedClaim.isNull()) {
-            Boolean value = emailVerifiedClaim.asBoolean();
-            isEmailVerified = value == null ? false : value;
-        }
-        Auth0UserInfo userInfo = new Auth0UserInfo(auth0Claims.get("email").asString(), auth0Claims.get("exp").asInt(),
-                isEmailVerified);
-        verifyUserConnection(auth0Claims.get("sub").asString(), userInfo.getEmail());
+    public Auth0UserInfo getAuth0UserInfo(@NonNull String idToken, String auth0Domain) throws DSMAuthenticationException {
+        try {
+            Map<String, Claim> auth0Claims = verifyAndParseAuth0TokenClaims(idToken, auth0Domain);
+            boolean isEmailVerified = false;
+            Claim emailVerifiedClaim = auth0Claims.getOrDefault("email_verified", null);
```

Comment:
do we care if email is verified?

Reply:
not really, we just extract it

Reply:
all dsm users are unverified, maybe we should change the auth0util to not care for that anymore

Reply:
I removed it